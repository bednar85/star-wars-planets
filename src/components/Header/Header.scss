.header {
  background-image: url('../../assets/imgs/rostyslav-zagornov-92gooayayc4.jpg');
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
  height: 25rem;
  padding: 1.25rem;
  position: relative;
  text-align: center;

  .app-logo {
    width: 12.5rem;
  }

  .image-attribution {
    bottom: 1.25rem;
    color: #fe5a6d;
    font-size: 0.625rem;
    font-style: italic;
    position: absolute;
    right: 1.25rem;

    a {
      color: #fe5a6d;
    }
  }
}
