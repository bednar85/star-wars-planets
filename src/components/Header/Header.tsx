import React, { ReactElement } from 'react';

import logo from 'assets/imgs/star-wars-logo-yellow.png';
import './Header.scss';

const Header = (): ReactElement => (
  <header className="header">
    <img className="app-logo" src={logo} alt="Star Wars" />
    <span className="image-attribution">
      Illustration by{' '}
      <a
        href="https://www.artstation.com/pabloolivera"
        target="_blank"
        rel="noreferrer noopener"
      >
        Pablo Olivera
      </a>
    </span>
  </header>
);

export default Header;
