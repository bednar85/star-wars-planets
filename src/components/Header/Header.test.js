import Header from './Header';

describe('App Component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper).toMatchSnapshot();
  });
});
