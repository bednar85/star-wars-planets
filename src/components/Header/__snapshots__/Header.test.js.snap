// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders correctly 1`] = `
<header
  className="header"
>
  <img
    alt="Star Wars"
    className="app-logo"
    src="star-wars-logo-yellow.png"
  />
  <span
    className="image-attribution"
  >
    Illustration by
     
    <a
      href="https://www.artstation.com/pabloolivera"
      rel="noreferrer noopener"
      target="_blank"
    >
      Pablo Olivera
    </a>
  </span>
</header>
`;
